html/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Welcome to my Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <div class="logo-container">
+      <a class="text-logo" href="#">Yassine</a>
+    </div>
+    <nav class="nav-menu">
+      <img src="/images/Union.png" alt="menu icon">
+    </nav>
+  </header>
+  <main class="headline">

```

Comment:
- [ ] You are making great use of the `<main>` tag but the headlines and the next sections should be in separated `<section>`s. So you can separate your headline, showcase, contact form, etc more easily to have a better structure. Please, wrap the headline inside a `<section>` tag.

It could look like:

![image](https://user-images.githubusercontent.com/81522435/138140393-61958937-3d28-47ff-9c56-ee02b4ca0b7a.png)